events/elections/2021/README.md

```diff
@@ -45,46 +45,79 @@ Eligibility for voting in 2021 is defined as:
+
+Examples of contributions that would be considered:
+* Slack admins who are not active in GitHub
+* Code of Conduct Committee members whose actions are private by default
+
+Examples of contributions that would NOT be considered:
+* Contributions to ecosystem projects and products
+* Organizing meetups or podcasts
 
 ### Schedule
 
+<!-- While finalizing the dates in the schedule, ensure that:
+- The Steering Committee and candidate Q+A occurs at a public SC meeting
+  (usually a Monday).
+- Dealine to submit voter exception forms and request a
+  replacement ballot is ~3 days before voting closes.
+- Private announcement of results to SC members is at least ~2 days
+  before private announcement to all candidates.
+- The interval between private announcement to all candidates and the
+  public announcement is a weekend.
+-->
+
 | Date         | Event                    |
 | ------------ | ------------------------ |
-| August XX    | Announcement of Election and publication of Voters.md |
+| July 1       | Steering Committee selects Election Committee |
+| August XX    | Announcement of Election and publication of voters.md |
 | August XX    | Steering Committee Meeting with Q+A with the candidates and community |
-| September XX | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September XX | All candidate bios due by 0000 UTC (5pm PST) |
 | ~1 week      | Election prep week (voters.md validation and CIVS setup and testing)
 | September XX | Election Begins via email ballots |
-| October XX   | Deadline to request a replacement ballot |
+| October XX   | Deadline to submit voter exception forms and request a replacement ballot |
 | October XX   | Election Closes by 0000 UTC (5pm PST) |
-| October XX   | Announcement of Results at Public Steering Committee meeting |
+| October XX   | Private announcement of Results to SC members not up for election |
+| October XX   | Private announcement of Results to all candidates |
+| October XX   | Public announcement of Results at Public Steering Committee Meeting |
+| October XX   | Election Retro |
 
 ## Candidacy Process
 
 **Nomination**
 
-If you want to stand for election, send an email to kubernetes-dev@googlegroups.com
-with the subject line "Steering Committee Nomination: Your Name (@yourgithub)".
+If you want to stand for the election, create an issue in this GitHub repo
+(kubernetes/community) with the title `Steering Committee Nomination: Your Name (@yourgithub)`.
+After creating the issue, send an email to kubernetes-dev@googlegroups.com
+with a link to the issue. The subject line of the email should be same as
+the title of the issue.
 
 If you want to nominate someone else, you may do so, but PLEASE talk to them
 first.
 
 If you wish to accept a nomination from someone else, reply to the nomination
-email saying something like "I accept the nomination".
+**issue** saying something like "I accept the nomination".
```

Comment:
we could also have the candidate PR that adds their bio file resolve the issue

Reply:
So ... I'm going to argue that we should leave this text alone BECAUSE this will all change when we deploy the new elections application, which has requirements for a specific git-centric workflow.